src/main/resources/application.yml

```diff
@@ -49,14 +49,10 @@ management:
     path-mapping:
       info: "status/isAlive"
   metrics:
-    export:
-      prometheus:
-        enabled: true
-    web:
-      server:
-        request:
-          autotime:
```

Comment:
Jeg vet ikke helt om vi har behov for autotime. 
"Should be applied at the ObservationRegistry level.", dvs satt gjennom bean